wgnlpy/wireguardpeer.py

```diff
@@ -27,7 +27,10 @@ def __init__(self, peer, spill_preshared_keys=False):
         self.endpoint = peer.get_attr('WGPEER_A_ENDPOINT')
         self.persistent_keepalive_interval = peer.get_attr('WGPEER_A_PERSISTENT_KEEPALIVE_INTERVAL')
         self.last_handshake_time = peer.get_attr('WGPEER_A_LAST_HANDSHAKE_TIME')
-        if not self.last_handshake_time > 0:
+        try:
+            if not self.last_handshake_time > 0:
```

Comment:
Instead of a try+except, you could also check if whether the variable has a None  value:

```python
if self.last_handshake_time is None or not self.last_handshake_time > 0:
```